all_test.go

```diff
@@ -25,32 +25,26 @@ Suggested test runs that you should do:
 	---- Without race detector ---
 
 	-- Test using maps/arrays mocking the backend:
-	go test github.com/IMQS/authaus -test.cpu 2
+	go test github.com/IMQS/authaus -test.cpu 2 -run TestAuth
```

Comment:
Generally, looks great.
This comment is a bit too long though. Rather don't duplicate all of the instructions with and without "-race". The user is smart enough to figure that out. Just leave the "-race" in all of the examples.